mewarpx/mewarpx/parallel_util.py

```diff
@@ -0,0 +1,27 @@
+from mpi4py import MPI as mpi
+import numpy as np
+
+
+from mewarpx.mwxrun import mwxrun
+
+
+comm_world = mpi.COMM_WORLD
```

Comment:
This is only used in the one place. Please inline it.

Reply:
I believe that there will be functions implemented in the future that also use comm_world, which is why it's defined globally.